### PR TITLE
Updated Bot.py (emote, shards etc)

### PR DESCRIPTION
**Pull request by 178619**
removed duplicated pet emotes, added shards info for stats command etc

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -841,12 +840,17 @@ async def on_message(self, message):
...
+				if channel.permissions_for(channel.guild.me).embed_links:
+					await Embed(
+						channel,
+						user=user,
+						title='Insufficent Permissions',
+						description='I do not have permissions to do this. Try enabling your DMS or giving me admin'
+					).send()
+			       else:
+					await channel.send(
+						'Insufficent Permissions: I do not have permissions to do this. Try enabling your DMS or giving me admin')
```
Comment (uwuplant) [inline bot.py:853]: 'Insufficent' should be 'Insufficient'.

File: bot.py [hunk 2]
```diff
@@ -1855,7 +1855,7 @@ async def stats(self, message, *args):
 			channel,
 			user=user,
 			title='Discord Stats',
-			description=f'```{server_rankings}```'
+			description=f'This Command was runned on shard {message.guild.shard_id if message.guild else 0} / {self.shard_count}\n```{server_rankings}```'
```
Comment (uwuplant) [inline bot.py:1858]: should be "This command was run on..."

**Comment by 178619** [comment]
mb

**Comment by 178619** [comment]
<s>word 'Insufficent' was from melon though</s>

**Comment by uwuplant** [comment]
looks good now!

**Pull request merged**